### PR TITLE
[Streams Platform] Issue with Templates overrides #4279

### PR DESCRIPTION
**Pull request by agustindidiego**
- Implement custom Twig Loader

### CODE REVIEW

**Comment by agustindidiego** [comment]
https://github.com/pyrocms/pyrocms/issues/4279

**Comment by Piterden** [comment]
https://github.com/anomalylabs/streams-platform/pull/374

File: src/StreamsServiceProvider.php [hunk 1]
```diff
@@ -379,6 +379,14 @@ function (Redirector $redirector) {
             return;
         }
 
+        $this->app->bind('twig.loader.viewfinder', function ($app) {
+            return $app->make('Rage\ViviencarnacionTheme\Twig\Loader', [
```
Comment (agustindidiego) [inline src/StreamsServiceProvider.php]: Oh f*ck, sorry about that. Let me fix that.

Comment (agustindidiego) [inline src/StreamsServiceProvider.php]: Fixed on https://github.com/anomalylabs/streams-platform/pull/372/commits/3068e99f36dadb7999c7258f113b60fee15185fc

**Comment by RyanThompson** [comment]
Excellent job! Many thanks ^_^

**Pull request merged**